### DISCUSSION TITLE
is it possible to change nextcloud mastercontainer interface port after installation?

### DISCUSSION THREAD

**Discussion by killtux** [nextcloud/all-in-one#3670] [Questions] [answered]
i changed the mastercontainer interface port from 8080:8080 to 8082:8080 in docker compose. in the admin settings the link is still pointing to 8080. looks like it doesnt change automatically. do i have to manually change it somewhere in a configuration file?

**Comment by szaimen** [comment, marked as answer]
> is it possible to change nextcloud mastercontainer interface port after installation?

Yes, just restart the containers from the AIO interface once.

**Reply by killtux** [reply]
great. thanks!
